update/process/metaBlock.go

```diff
@@ -106,7 +118,7 @@ func (m *metaBlockCreator) CreateNewBlock(
 		PubKeysBitmap:          []byte{1},
 	}
 
-	return metaHdr, blockBody, nil
+	return metaHdr, nil
```

Comment:
here we must add the unprocessed miniblock headers.

Reply:
There will be no unprocessed miniblocks, please see the explanation below.